frontend/packages/dev-console/src/utils/guided-tour.ts

```diff
@@ -0,0 +1,50 @@
+import { TourDataType } from '@console/app/src/components/tour';
+
+const getSelector = (id: string): string => `[data-tour-id="${id}"]`;
+
+export const getGuidedTour = (): TourDataType => ({
+  intro: {
+    heading: 'Welcome to Dev Perspective!',
+    content: 'Dummy',
+    nextButtonText: 'Get Started',
+    backButtonText: 'Close',
```

Comment:
These should be the defaults.
I don't think we need to let them change the back button text.